locales/de.json

```
@@ -2205,6 +2211,12 @@
 				"theNextQuestionnaireIsOnline": "Die nächste Befragung zum Projekt {{shortTitle}} ist online. Gerne möchten wir Sie erneut um Ihre Mitarbeit bitten! Die Bearbeitung wird ca. 20 - 25 Minuten Ihrer Zeit in Anspruch nehmen."
 			}
 		},
+		"form_ChangeLanguage" : {
+			"label": "Sprache",
```

Keys should follow this schema: `$Location.$Type.$Description`
```suggestion
			"label": {
				"language" : "Sprache"
			},
```

---

done ;)